app/src/main/res/layout/fragment_pin_view.xml

```diff
@@ -156,5 +167,70 @@
+            android:adjustViewBounds="false"
+            android:background="@drawable/ic_circle"
+            android:backgroundTint="?attr/colorSecondary"
+            android:clickable="true"
+            android:focusable="true"
+            android:foreground="@drawable/ic_add_comment"
+            android:foregroundTint="?attr/colorOnSecondary"
+            android:scaleType="fitCenter"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
     </androidx.constraintlayout.widget.ConstraintLayout>
+
+            <ImageView
+                android:id="@+id/send_comment_button"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:adjustViewBounds="false"
+                android:background="@drawable/ic_circle"
+                android:backgroundTint="?attr/colorPrimary"
+                android:clickable="true"
+                android:focusable="true"
+                android:foreground="@drawable/ic_send"
+                android:foregroundTint="?attr/colorOnSecondary"
+                android:scaleType="fitCenter"
+                android:layout_marginRight="18dp"
+                android:visibility="gone"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintBottom_toBottomOf="@+id/comment_input_layout"
+                app:layout_constraintTop_toTopOf="@+id/comment_input_layout" />
+
+            <com.google.android.material.textfield.TextInputLayout
+                android:layout_marginTop="10dp"
+                android:id="@+id/comment_input_layout"
+                style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintTop_toBottomOf="@+id/pin_info_layout"
+                android:layout_marginHorizontal="5dp"
+                android:paddingRight="80dp"
+                android:visibility="gone">
+
+                <com.google.android.material.textfield.TextInputEditText
+                    android:id="@+id/comment_edittext_layout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:importantForAutofill="no"
+                    android:hint="@string/add_a_comment" />
+            </com.google.android.material.textfield.TextInputLayout>
+
+            <androidx.recyclerview.widget.RecyclerView
+                android:id="@+id/comment_recycler_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginTop="10dp"
+                android:paddingBottom="140dp"
+                app:layout_constraintTop_toBottomOf="@+id/comment_input_layout" />
```

Comment:
Let's add a horizontal margin of 10dp to make this match the other parts of the UI.